### PR TITLE
[Tests-Only] Modified API steps to always mention username

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR:
- API steps made not to use `client.globals.currentUser` rather always mention `username` inside the step

## Related Issue

- Fixes https://github.com/owncloud/phoenix/issues/3946

## How Has This Been Tested?

- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/stepDefinitions/sharingContext.js [hunk 1]
```diff
@@ -481,13 +481,6 @@ Given('user {string} has shared file/folder {string} with user {string}', functi
   return shareFileFolder(elementToShare, sharer, receiver)
 })
 
-Given('the user has shared file/folder {string} with user {string}', function(
-  elementToShare,
-  receiver
-) {
-  return shareFileFolder(elementToShare, client.globals.currentUser, receiver)
-})
-
```
Comment (kiranparajuli589) [inline tests/acceptance/stepDefinitions/sharingContext.js:490]: Removed because not used in any feature

**Review by individual-it — APPROVED** [review]

File: tests/acceptance/features/webUITrashbin/trashbinFilesFolders.feature [hunk 2]
```diff
@@ -28,7 +28,7 @@ Feature: files and folders exist in the trashbin after being deleted
 
   @skipOnOCIS @ocis-reva-issue-111
   Scenario: Delete a file with problematic characters and check it is in the trashbin
-    Given the user has renamed the following files
+    Given the user "user1" has renamed the following files
```
Comment (phil-davis) [inline tests/acceptance/features/webUITrashbin/trashbinFilesFolders.feature:31]:
```suggestion
    Given user "user1" has renamed the following files
```

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 3]
```diff
@@ -830,19 +836,16 @@ Then('the deleted elements should be listed on the webUI', function() {
   return assertDeletedElementsAreListed()
 })
 
-Given('the user has renamed the following files', function(table) {
+Given('the user {string} has renamed the following files', function(userId, table) {
```
Comment (phil-davis) [inline tests/acceptance/stepDefinitions/filesContext.js:839]:
```suggestion
Given('user {string} has renamed the following files', function(userId, table) {
```

File: tests/acceptance/features/webUITrashbin/trashbinFilesFolders.feature (hunk 2 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/webUITrashbin/trashbinFilesFolders.feature]: done!

File: tests/acceptance/stepDefinitions/filesContext.js (hunk 3 above)
Comment (kiranparajuli589) [reply tests/acceptance/stepDefinitions/filesContext.js]: done!

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/phoenix/10903/25/19
```
   ? Given the user has created folder "new-lorem.txt"
       Undefined. Implement with the following snippet:

         Given('the user has created folder {code}', function (code) {
         // Given('the user has created folder {string}', function (string) {
           // Write code here that turns the phrase above into concrete actions
           return 'pending';
         });
```
This step needs adjusting.

**Pull request merged**